src/sage/rings/qqbar.py

```diff
@@ -730,7 +730,7 @@ def _factor_multivariate_polynomial(self, f, proof=True):
         REFERENCES:
 
         - [GCL1992]_ Section 8.8
-        - https://trac.sagemath.org/attachment/ticket/25390/qqbar.pdf
+        - :trac:`25390` and https://github.com/sagemath/sage-prod/files/10659152/qqbar.pdf.gz
 
```

Comment:
Drop `-prod`

Reply:
@mkoeppe On the other side, could we remove `-prod` in the attachment links in the converted issues?

Reply:
That's now https://github.com/sagemath/trac-to-github/issues/188